instrumentation/reactor-3.1/library/src/main/java/io/opentelemetry/instrumentation/reactor/TracingOperator.java

```diff
@@ -44,6 +47,13 @@ public static TracingOperatorBuilder newBuilder() {
     return new TracingOperatorBuilder();
   }
 
+  private static final ContextKey<Context> TRACE_CONTEXT_KEY =
+      ContextKey.named("otel-trace-context");
+
+  // have to be re-initialized after hooks are set/reset
+  private static Mono<String> dummyMono = Mono.just("");
+  private static Flux<String> dummyFlux = Flux.just("");
```

Comment:
Unfortunately, they have to be updated when hook is registered/reset, otherwise, they will always make `runInScope` 
 populate current from the context